alica_engine/CMakeLists.txt

```diff
@@ -4,56 +4,104 @@ project(alica_engine)
 add_compile_options(-std=c++17)
 set(CMAKE_BUILD_TYPE Debug)
 
+# Set up utils to allow Colcon/catkin to find the packages
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME}/cmake) 
+
+
 ##Yaml
 find_package(yaml-cpp REQUIRED)
 
-##ROS/Catkin
-find_package(catkin QUIET)
-if (catkin_FOUND)
-  find_package(catkin REQUIRED COMPONENTS fsystem alica_common_config alica_solver_interface)
-else (catkin_FOUND)
-  set(catkin_LIBRARIES "fsystem" "alica_common_config" "alica_solver_interface")
-endif(catkin_FOUND)
-
-if(catkin_FOUND)
-  catkin_package(
-    INCLUDE_DIRS include
-    LIBRARIES alica_engine
-    CATKIN_DEPENDS fsystem alica_common_config alica_solver_interface
-  )
-endif(catkin_FOUND)
+# Imports
+find_package(fsystem REQUIRED)
+find_package(alica_common_config REQUIRED)
+find_package(alica_solver_interface REQUIRED)
+
 
 ###########
 ## Build ##
 ###########
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${YAML_CPP_INCLUDE_DIR})
+include_directories(include 
```

Comment:
Removed all the ones that it let me - since alica_common_config is just headers it seems like it still needs it in the directories since it has no compiled file to link against